node-src/git/getParentCommits.test.ts

```diff
@@ -397,7 +397,7 @@ describe('getParentCommits', () => {
     const mockIndex = createMockIndex(repository, [['A', 'main']]);
     const mockIndexWithNullFirstBuildCommittedAt = (queryName, variables) => {
       if (queryName === 'FirstCommittedAtQuery') {
-        return { app: { firstBuild: { committedAt: null } } };
+        return { app: { firstBuild: {} } };
```

Comment:
I think GraphQL will return an explicit `null` here, should we encode that over our lint rule?

Reply:
Ope! Good callout, I'll adding this back in.